charts/logging-apps/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: logging-apps
 description: Argo CD app-of-apps config for logging applications
 type: application
-version: 0.35.0
+version: 0.35.1
```

Comment:
imo this should be a minor version, as it's a potentially breaking change

Reply:
I updated this.

Users should keep in mind that the `-apps` charts have been doing https://0ver.org/ without any goal to change that.